share/jupyterhub/static/less/admin.less

```diff
@@ -1,3 +1,9 @@
 i.sort-icon {
   margin-left: 4px;
 }
+
+.version_footer {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+}
```

Comment:
I think this one caused the build to fail during the lint execution.

https://travis-ci.org/jupyterhub/jupyterhub/jobs/597187918

Reply:
Oh, thank you! I was only watching the 3.6 tests and forgot to check back in on the others. I'll do some fiddling.